pkg/pack.sh

```diff
@@ -48,7 +48,7 @@ else
     init_distro_name
 fi
 
-__common_parameters="/p:$__targets_param /p:DistroRid=$__distro_rid /verbosity:minimal"
+__common_parameters="/p:$__targets_param /p:DistroRid=$__distro_rid /verbosity:minimal /p:Platform=x64"
```

Comment:
This is a band-aid that shouldn't have been checked-in in its current form.

I added it because the `pack.sh` script on Unix was computing package names as `runtime.ubuntu.14.04-linux.system.io.compression`. It was relying on the `Platform` and `PackagePlatform` variables to be set and was defaulting them oddly when they weren't being set. I'll look into it more and get a better solution or at least figure out why it's failing that way.